File: src/2413-Smallest-Even-Multiple/smallest_even_multiple.py
class Solution:
    def smallestEvenMultiple(self, n: int) -> int:
        if n % 2 == 0:
            return n
        else:
            return n * 2

if __name__ == "__main__":
    n: int = 5

    solution = Solution()

    print(solution.smallestEvenMultiple(n))

File: src/2413-Smallest-Even-Multiple/smallest_even_multiple.cpp
#include <iostream>

class Solution {
public:
    int smallestEvenMultiple(int n) {
        if (n % 2 == 0) return n;
        else return n * 2;
    }
};

int main() {
    int n = 5;

    Solution solution;

    std::cout << solution.smallestEvenMultiple(n); 

    return 0;
}